src/livox_points_plugin.cpp

```diff
@@ -135,13 +140,22 @@ void LivoxPointsPlugin::OnNewLaserScans() {
         sensor_msgs::PointCloud2Iterator<float> out_z(scan_point, "z");
 
         const size_t raySize = samplesStep;
+        size_t countNoReturn {0};
         for (const auto [idx, rotateInfo] : points_pair) {
             auto range = RangeMin() + rayShape->GetRange(idx);
             auto intensity = rayShape->GetRetro(idx);
             if (range >= RangeMax()) {
                 range = 0.0;
             } else if (range <= RangeMin()) {
                 range = 0.0;
+                // this one should could be a point with no return -> publish it in second cloud with zenith / azimuth
```

Comment:
what do you mean by this comment? or is it referring to the next block where retro == 0.0?

Reply:
yes, but it's a unnecessary comment, will remove it.